ui.apps/src/main/content/jcr_root/apps/saas-aem-module/i18n/en.json

```diff
@@ -3,5 +3,12 @@
   "com.valtech.aem.saas.core.search.submit.button.label": "Search",
   "com.valtech.aem.saas.core.search.suggestion.text": "Did you mean:",
   "com.valtech.aem.saas.core.search.input.placeholder.text": "Type search term here...",
-  "com.valtech.aem.saas.core.search.noResults.text": "No results found."
+  "com.valtech.aem.saas.core.search.noResults.text": "No results found.",
+  "com.valtech.aem.saas.core.search.apiConnectionFail.text": "Can not communicate to Search API.",
+  "com.valtech.aem.saas.core.search.apiConnectionFail.furtherAction.checkLogFiles.text": "Check log files.",
+  "com.valtech.aem.saas.core.search.apiConnectionFail.furtherAction.checkOsgiConfigurations.text": "Check OSGi Configurations.",
+  "com.valtech.aem.saas.core.search.caConfiguration.failedToResolveConfigurationMessage": "Failed to resolve CA Search Configuration for resource: {}",
+  "com.valtech.aem.saas.core.search.caConfiguration.indexRequiredMessage": "Search Index must be configured.",
+  "com.valtech.aem.saas.core.search.caConfiguration.clientRequiredMessage": "Search Client must be configured.",
+  "com.valtech.aem.saas.core.search.caConfiguration.projectRequiredMessage": "Project Id must be configured and should be a non-negative integer."
```

Comment:
Any chance that we default to 1 to avoid this error message? As I understood the SaaS Team 1 is always present